docs/development/visualize/development-create-visualization.asciidoc

```diff
@@ -335,7 +336,36 @@ This response matches the visData property of the <visualization> directive.
 
 [[development-default-response-handler]]
 ==== default response handler
-Default response handler will convert pure elasticsearch response to tabular format.
+Default response handler will convert pure elasticsearch response to tabular format. Its the recommended response
+handler to use. The response object will contain a table property, which is an array of all the tables produced (if 
+you use the split chart for example). Each of the table objects has two properties:
+
```

Comment:
`The response object contains a table property, which is an array of all the tables in the response. Each of the table objects has two properties:`